DOCS/man/input.rst

```diff
@@ -1937,10 +1937,10 @@ Property list
     Estimated deviation factor of the vsync duration.
 
 ``video-aspect`` (RW)
-    Video aspect, see ``--video-aspect``.
+    Video aspect, sse ``--video-aspect-override``.
 
     If video is active, this reports the effective aspect value, instead of
-    the value of the ``--video-aspect`` option.
+    the value of the ``--video-aspect-override`` option.
```

Comment:
Does this sentence really still make sense?

It should also mention that it's deprecated, and how a user should change to non-deprecated mechanisms. This is the nasty part because you need to think about how human users and API users might have (mis)used it.

There's also another mention of ``video-aspect`` in input.rst in the "Inconsistencies between options and properties" which this commit touches upon.

Reply:
Clarified the interaction in the documentation of `video-aspect`. I also just removed that reference since there's no more `--video-aspect` option, technically.